src/main/java/com/inventorysystem/inventory/dto/Product.java

```diff
@@ -3,10 +3,10 @@
 
 public @Data
 class Product {
-    private int ProductID;
-    private String ProductName;
-    private String ProductDesc;
-    private int VendorID;
-    private double Price;
-    private int Quantity;
+    private int productID;
+    private String productName;
```

Comment:
Naming conventions according to the Java standards

Reply:
Good.